### PR TITLE
NAS-128455 / 24.10 / Clean up and copy over disk details card from old enclosure module

### PR DESCRIPTION
**Pull request by bvasilenko**
**Testing**
Verify new disk details UI block (see ticket for more details)

### CODE REVIEW

**Review by RehanY147 — CHANGES_REQUESTED** [review]
The buttons like "Show Pools" should have redirection to the storage pools page.

**Review by RehanY147 — APPROVED** [review]
Ignore the previous review. The button is only supposed to show the disks layout on the enclosure and not redirect to the storage page.

**Review by RehanY147 — CHANGES_REQUESTED** [review]
image
State should have `N/A` if not available.

File: src/app/pages/system/enclosure/components/views/enclosure-view/disks-overview/disks-overview.component.html [hunk 1]
```diff
@@ -41,25 +41,44 @@
...
+          <div class="label">{{ 'Adv. Power Mgmt.' | translate }}</div>
+          <div class="value">{{ 'N/A' | translate }}</div>
+        </div>
+        <div class="detail">
+          <div class="label">{{ 'S.M.A.R.T. Enabled' | translate }}</div>
+          <div class="value">{{ 'N/A' | translate }}</div>
+        </div>
+        <div class="detail">
+          <div class="label">{{ 'S.M.A.R.T. Options' | translate }}</div>
+          <div class="value">{{ 'N/A' | translate }}</div>
```
Comment (RehanY147) [inline src/app/pages/system/enclosure/components/views/enclosure-view/disks-overview/disks-overview.component.html:64]: Why is this value hard coded? The response from `webui.enclosure.dashboard` is supposed to have all the disk details that we need for our enclosure.

**Review by RehanY147 — CHANGES_REQUESTED** [review]

File: src/app/pages/system/enclosure/components/views/enclosure-view/disks-overview/disks-overview.component.html (hunk 1 above)
Comment (undsoft) [reply src/app/pages/system/enclosure/components/views/enclosure-view/disks-overview/disks-overview.component.html:64]: and if this is not something that is available in `webui.enclosure.dashboard`, it's okay to remove it from the UI.

**Review by RehanY147 — APPROVED** [review]

**Pull request merged**